ports/quake.2/gameinfo.xml

```diff
@@ -3,13 +3,44 @@
   <game>
     <path>./Quake 2.sh</path>
     <name>Quake II</name>
-    <desc>A sequel in name only, this follow up to id's 1996 Quake, is a first-person shooter revolving around the Earth-Strogg war. The Strogg alien race threatens the Earth, but Earth has launched a counter-offensive, Operation Alien Overlord. Overlord takes the battle to the Strogg homeworld with the ultimate objective of securing their capital city and killing their ruler. 
- 
-As a FPS, Quake II contains a variety of weapons including the fall back Blaster, the Railgun, and the secret BFG10K. Health, armor, and ammunition pick-ups can be found during levels, as can the classic Quake's Quad Damage power-up. Game play involves mowing down alien opponents as the player progresses through levels, completing in-game objectives. Quake II also comes with multiplayer support for Deathmatches.</desc>
-    <releasedate>19971201T000000</releasedate>
+    <desc>A sequel in name only, this follow up to id's 1996 Quake, is a first-person shooter revolving around the Earth-Strogg war. The Strogg alien race threatens the Earth, but Earth has launched a counter-offensive, Operation Alien Overlord. Overlord takes the battle to the Strogg homeworld with the ultimate objective of securing their capital city and killing their ruler. As a FPS, Quake II contains a variety of weapons including the fall back Blaster, the Railgun, and the secret BFG10K. Health, armor, and ammunition pick-ups can be found during levels, as can the classic Quake's Quad Damage power-up. Game play involves mowing down alien opponents as the player progresses through levels, completing in-game objectives. Quake II also comes with multiplayer support for Deathmatches.</desc>
+    <image>./quake2/cover.png</image>
```

Comment:
cover.png is missing

Reply:
I don't see any license file, isn't there any to include ?

Reply:
I didn't delete it nor have I changed the cover.png file nor the screenshot.jpg image.  I'm guessing they're not showing since I haven't altered them.  I'm not sure about the license file.  The original uploader didn't upload one, either.  I don't mind seeing if there is one for Yamagi Q2, but at the very least the link to the github source is in the README.  Also, the thing that's bothering me the most is that I deleted the temp folder, but it's still showing up in this PR despite my removing it.

Reply:
Yamagi Quake II LICENSE added

Reply:
To clarify and reiterate, the cover.png is indeed already in the repo for this port of Quake II.

Reply:
> To clarify and reiterate, the cover.png is indeed already in the repo for this port of Quake II.

Oh yeah sorry I missed it !